app/tests/unit/components/Form/ProjectMilestoneReportForm.test.tsx

```diff
@@ -184,13 +191,7 @@ describe("The ProjectMilestoneReportForm", () => {
       require("lib/helpers/validateFormWithErrors"),
       "default"
     );
-    // resolve the update operations, else 'isUpdating' will disable the submit button
-    componentTestingHelper.environment.mock.resolveMostRecentOperation({
-      data: { variables: { input: "asdf" } },
-    });
-    componentTestingHelper.environment.mock.resolveMostRecentOperation({
-      data: { variables: { input: "asdf" } },
-    });
```

Comment:
I took this out because 1) once I updated the default query to have the milestone fields, these tests failed, 2) none of the other report tests use these lines